guides/common/modules/services.adoc

```diff
@@ -0,0 +1,92 @@
+[[Services]]
+= Services
+
+{ProjectName} uses a set of back-end services to perform the actual job. The status of these services can negatively influence the whole system and it’s one of the first things to check when some errors occur.
+
+==== Retrieve status of services
+
+The status of back-end services can be found either from the Web UI on the /about page
+
+* The `hammer ping` command can be used to get information from the database and {ProjectName} services.
+[source,shell,options="nowrap"]
+----
+$ hammer ping
+database:
+    Status:          ok
+    Server Response: Duration: 0ms
+candlepin:
+    Status:          ok
+    Server Response: Duration: 869ms
+candlepin_events:
+    Status:          ok
+    message:         0 Processed, 0 Failed
+    Server Response: Duration: 0ms
+candlepin_auth:
+    Status:          ok
+    Server Response: Duration: 102ms
+katello_events:
+    Status:          ok
+    message:         0 Processed, 0 Failed
+    Server Response: Duration: 1ms
+pulp3:
+    Status:          ok
+    Server Response: Duration: 298ms
+foreman_tasks:
+    Status:          ok
+    Server Response: Duration: 8ms
+----
+
+* Foreman Maintain can be used to check the status of the services running in systemd.
+[source,shell,options="nowrap"]
+----
+$ foreman-maintain service status
+Running Status Services
+================================================================================
+Get status of applicable services:
+rh-mongodb34-mongod, postgresql, qdrouterd, qpidd, rh-redis5-redis, squid, pulp_celerybeat, pulp_resource_manager, pulp_streamer, pulp_workers, smart_proxy_dynflow_core, tomcat, dynflow-sidekiq@orchestrator, foreman, httpd, puppetserver, dynflow-sidekiq@worker, dynflow-sidekiq@worker-hosts-queue, foreman-proxy
```

Comment:
Should we delete the reference to mongo?

Reply:
And celery?

Reply:
Yeah, all the Pulp2 stuff should be gone from examples